lib/ruby_indexer/lib/ruby_indexer/index_rbs.rb

```diff
@@ -0,0 +1,45 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyIndexer
+  class IndexRbs
```

Comment:
The name `IndexRbs` makes it sound like there are two indices: one for RBS and the other one.

Can we call this indexer, collector or something like that?

Reply:
Let's use this occasion to rectify the naming, I think we should have one `Index`, the data structure, and a bunch of `XIndexer`, the things collecting data to populate the index: `RubyIndexer`, `RBSIndexer`, `RBIIndexer`, etc.

Reply:
I've renamed it to `RubyIndexer`.

@Morriar I agree with your suggestion, but we can do that in a follow-up?

Reply:
Yep 👍